src/views/Error/Error.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { i18n } from '../../config'
+
+class Error extends React.Component {
+  render() {
+    return (
+      <div className="login eapp-core" id="login">
+        <div id="seal-header" className="seal-header text-center">
+          <div className="content">
+            <img
+              src="/img/nbis-seal.png"
+              alt="National Background Investigation Services"
+            />
+            <h2>{i18n.t('login.title')}</h2>
+          </div>
+        </div>
+        <div className="content">
+          <div className="table one">
+            <div id="error" className="auth error">
+              <h3>{i18n.t('application.loading.error.title')}</h3>
+              {i18n.m('application.loading.error.para')}
+              <a href="/" title={i18n.t('application.loading.error.button')}>
+                {i18n.t('application.loading.error.button')}
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+Error.defaultProps = {}
+
+function mapStateToProps(state) {
```

Comment:
I would probably just remove all of the `connect` code from this file and the test since it doesn't seem to be used